Microsoft.Toolkit.Uwp.UI.Controls.Primitives/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Resources;
+using System.Runtime.CompilerServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: InternalsVisibleTo("UnitTests.UWP")]
+[assembly: InternalsVisibleTo("UnitTests.XamlIslands.UWPApp")]
```

Comment:
I don't know much about how XamlIslands works, do you think we need this for the primitives, @michael-hawker?

Reply:
This is just exposing it to the XamlIslands tests, so we probably need to tweak those and the Unit Tests as well if they touch these controls?

Reply:
It seems like the xmal islands test worked just fine without that line.